src/tree/mod.rs

```diff
@@ -84,7 +84,7 @@ where
         // Sync the identity tree to the chain tip, also updating the chain_state with the latest roots on all chains
         tracing::info!("Syncing to head");
         self.sync_to_head().await?;
-        tracing::info!("Synced to head in {:?} seconds", start_time.elapsed());
+        tracing::info!("Synced to head in {:?}", start_time.elapsed());
```

Comment:
Consider this

```suggestion
        tracing::info!(sync_time = start_time.elapsed(), "Synced to head" );
```

this way we can extract this data in logs for our dashboards

Reply:
Although in that case you'd probably want to explicitly use https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_millis